modules/javafx.controls/src/main/resources/com/sun/javafx/scene/control/skin/modena/modena.css

```diff
@@ -357,6 +357,9 @@ is being used to size a border should also be in pixels.
     /** Focus line for keyboard focus traversal on cell based controls */
     -fx-cell-focus-inner-border: derive(-fx-selection-bar,30%);
 
+    /* Cell border color is initialized here to avoid a race condition during initialization */
```

Comment:
I would still instead write something like `/**Border color for table cells. */` and _maybe_ link the JDK ticket. But developers looking in modena.css are more interested what the variables do than why they are there. But open for other opinions